src/gax-internal/src/options.rs

```diff
@@ -14,6 +14,7 @@
 
 pub use auth::credentials::Credential as Credentials;
 
+#[allow(rustdoc::broken_intra_doc_links)]
 /// The client configuration for [crate::http::ReqwestClient] and [crate::grpc::Client].
```

Comment:
Or we can change the comments to not be `rustdoc` comments: 

```suggestion
// The client configuration for [crate::http::ReqwestClient] and [crate::grpc::Client].
```